src/openassetio-core/src/hostApi/ManagerFactory.cpp

```diff
@@ -121,11 +122,17 @@ ManagerPtr ManagerFactory::defaultManagerForInterface(
     Str msg = "Could not load default manager config from '";
     msg += configPath;
     msg += "', file does not exist.";
-    throw std::runtime_error(msg);
+    throw errors::InputValidationException(msg);
```

Comment:
[plan] Docs need updating (and for below).

Reply:
Addressed as part of #1104